picard/formats/__init__.py

```diff
@@ -84,107 +84,6 @@ def open(filename):
         return None
 
 
-def _insert_bytes_no_mmap(fobj, size, offset, BUFFER_SIZE=2**16):
```

Comment:
Imho, this removal should go in another PR. Perhaps i miss something but it doesn't look it is directly related to PICARD-933.

@samj1912 : can you create another ticket and another PR for the monkey patching removal ?